EU4toV2/Source/Mappers/Ideas/IdeaEffects.h

```diff
@@ -39,48 +39,53 @@ class IdeaEffects: commonItems::parser
 	public:
 		IdeaEffects(std::istream& theStream);
 
-		int getArmyInvestmentValue() const { return armyInvestmentValue; }
-		int getNavyInvestmentValue() const { return navyInvestmentValue; }
-		int getCommerceInvestmentValue() const { return commerceInvestmentValue; }
-		int getCultureInvestmentValue() const { return cultureInvestmentValue; }
-		int getIndustryInvestmentValue() const { return industryInvestmentValue; }
-
-		double getArmyTechScoreValue() const { return armyTechScoreValue; }
-		double getNavyTechScoreValue() const { return navyTechScoreValue; }
-		double getCommerceTechScoreValue() const { return commerceTechScoreValue; }
-		double getCultureTechScoreValue() const { return cultureTechScoreValue; }
-		double getIndustryTechScoreValue() const { return industryTechScoreValue; }
-
-		double getUpperHouseLiberalValue() const { return upperHouseLiberalValue; }
-		double getUpperHouseReactionaryValue() const { return upperHouseReactionaryValue; }
-
-		int getOrderInfluenceValue() const { return orderInfluenceValue; }
-		int getLibertyInfluenceValue() const { return libertyInfluenceValue; }
-		int getEqualityInfluenceValue() const { return equalityInfluenceValue; }
-
-		std::set<int> getLiteracyLevels() const { return literacyLevels; }
+		int getArmy() const { return army; }
+		int getNavy() const { return navy; }
+		int getCommerce() const { return commerce; }
+		int getCulture() const { return culture; }
+		int getIndustry() const { return industry; }
+
+		int getOrder() const { return order; }
+		int getLiberty() const { return liberty; }
+		int getEquality() const { return equality; }
+		int getLiteracy() const { return literacy; }
+
+		int getReactionary() const { return reactionary; }
+		int getLiberal() const { return liberal; }
+
+		int getSlavery() const { return slavery; }
```

Comment:
Make sure there are tests for all new items.

Reply:
all tests rewritten and renamed.